### PR TITLE
[Homework #23] - Testing by Ivan Ishchenko

### PR DESCRIPTION
**Pull request by ViVse**

### CODE REVIEW

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
Hello @ViVse!
Please add tests for other components, not only for App.js.
E.g: for Input.js, Task.js.

**Comment by ViVse** [comment]
@VitalikMelnyk, added more tests

File: homeworks/ivan-ishchenko/23-testing/src/components/Input.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import {render, fireEvent, screen} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import Input from './Input';
+
+describe('Input', () => {
+  it('renders with default props', () => {
+    render(<Input />);
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/23-testing/src/components/Input.test.js:8]: Please fix this console error.
Screenshot 2023-10-19 at 17 49 00

File: homeworks/ivan-ishchenko/23-testing/src/components/Task.test.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
+import React from 'react';
```
Comment (VitalikMelnyk) [inline homeworks/ivan-ishchenko/23-testing/src/components/Task.test.js:1]: Please, fix this error in console.
Screenshot 2023-10-19 at 17 50 52

**Review by VitalikMelnyk — CHANGES_REQUESTED** [review]
Hello @ViVse!
I have checked your PR and left comments.

**Comment by ViVse** [comment]
@VitalikMelnyk, fixed errors

**Review by VitalikMelnyk — APPROVED** [review]
Approved.

**Pull request merged**